tests/test_symbolics.py

```diff
@@ -94,3 +97,139 @@ def test_solve_time():
     # TODO: replace by retreive_derivatives after Fabio's PR
     assert sympy.simplify(u.dx - sol.args[2].args[0].args[1]) == 0
     assert sympy.simplify(sol - (-dt**2*u.dx/m + 2.0*u - u.backward)) == 0
+
+
+class Test_relations_w_assumptions(object):
```

Comment:
blank line after class definition

Reply:
and in python class names are camel case. No underscores, please

Reply:
Done